lisa/tools/ip.py

```diff
@@ -73,6 +80,8 @@ def restart_device(self, nic_name: str) -> None:
                 f"fail to restart [down then up] the nic {nic_name}"
             ),
         )
+        if persist:
+            self.node.tools[Startup].add_command(f"ip link set dev {nic_name} up")
```

Comment:
What's the reason to add it for persist? My understanding is that the nics will be up by default.

Reply:
We need to put nic in up state after adding the ip address. I think it's better to explicitly call it rather than replying on distro to do it. This is something that was followed in v2 as well.

Reply:
Let's check if it's supported by distro by default. If yes, it doesn't need this logic. The unnecessary steps bring other problems.

Reply:
Sure! I will validate this and let you know

Reply:
ok. BTW, if this test case is failed to recover the nodes, the node should be marked as dirty.

Reply:
Sure! I will up update this